tor_archivist/main.py

```diff
@@ -54,86 +39,61 @@ def noop(cfg):
 
 def run(cfg):
     if not CLEAR_THE_QUEUE_MODE and cfg.sleep_until >= time.time():
+        # TODO: if ctq is active, send ctq query parameter to expired endpoint
```

Comment:
Are we planning on completing this in the PR or waiting until later to implement?

Reply:
I'm fine with implementing this later